packages/client/hmi-client/src/components/workflow/ops/optimize-ciemss/tera-optimize-ciemss-node.vue

```diff
@@ -139,6 +141,30 @@ const startForecast = async (simulationIntervetions) => {
 	return makeForecastJobCiemss(simulationPayload, nodeMetadata(props.node));
 };
 
+const preparedCharts = computed(() => {
+	const state = props.node.state;
+	const preForecastRunId = state.preForecastRunId;
+	const postForecastRunId = state.postForecastRunId;
+	if (!postForecastRunId || !preForecastRunId) return [];
+	const preResult = runResults.value[preForecastRunId];
+	const preResultSummary = runResultsSummary.value[preForecastRunId];
+	const postResult = runResults.value[postForecastRunId];
+	const postResultSummary = runResultsSummary.value[postForecastRunId];
+	return state.selectedSimulationVariables.map((variable) =>
+		createOptimizeForecastChart(preResult, preResultSummary, postResult, postResultSummary, [], {
+			width: 150,
+			height: 120,
+			variables: [pyciemssMap[variable]],
+			statisticalVariables: [`${pyciemssMap[variable]}_mean`],
+			legend: false,
+			groupField: 'sample_id',
+			timeField: 'timepoint_id',
+			xAxisTitle: '',
+			yAxisTitle: ''
```

Comment:
Why those options are not done by `createOptimizeForecastChart`?
Are we going to style _Optimize Forecast_ chart differently through the application?

Reply:
Yup it can be styled different, this is more of a condensed version for when we view the chart from the node view vs a bigger more detailed one within the drilldown